File: Project2/Sheep.cpp
#include "Sheep.h"
#include "World.h"
#include <stdlib.h>

Sheep::Sheep(const Vector2& pos) : Animal(pos, 4, 4, 3, 'S')
{
}

Sheep::~Sheep()
{
}

Sheep* Sheep::Clone(const Vector2& pos) const
{
	return new Sheep(pos);
}

std::string Sheep::GetName() const
{
	return "Sheep";
}

File: Project2/WorldSimulator/Animals/Sheep.java
package WorldSimulator.Animals;

import WorldSimulator.Animal;
import WorldSimulator.Vector2;

import java.awt.*;

public class Sheep extends Animal
{
    public Sheep(Vector2 pos)
    {
        super(pos, 4, 4, 3, 'S', new Color(255, 255, 255));
    }

    @Override
    public String toString()
    {
        return "Sheep";
    }
}
